platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 5.10.0-alpha.1
+Starting this release, the Mapbox Maps SDK for iOS will use a pre-built GL Native binary licensed under the [Mapbox Terms of Service](https://www.mapbox.com/legal/tos). The license of the Maps SDK for iOS remains BSD-2. For more information on using this and future releases, see [README.md](https://github.com/mapbox/mapbox-gl-native-iOS/blob/master/README.md). For more details on licensing, see [LICENSE.md](https://github.com/mapbox/mapbox-gl-native-iOS/blob/master/LICENSE.md).
+
+### ✨ New features
+
+- Source Modularization.
+  Introduces a SourceManager + *SourceFactory classes similar to the work done on layer modularization.
+
+- [core] Enable source "volatile" flag initialization in style.
+
+  The source `volatile` flag value can be defined in the style as following:
+
+  ```
+  "example_source": {
+    "volatile": true
+  }
+  ```
+
+  For the sources with the `volatile` flag set, the tiles are not stored in the local storage. The `volatile` flag is applicable for all source types, which allow loading tiles from a remote server i.e. `vector`, `raster`, `raster-dem` source types. For other source types this setting is ignored.
+
+- [core] Enable `Renderer::collectPlacedSymbolData()` API for all map modes.
+
+  Before, it was functional only for the `Tile` map mode.
+
+- [core] Add `emphasis-circle-color` and `emphasis-circle-radius` properties to LocationIndicator layer.
+
+  LocationIndicator layer gets the ability to draw an additional circle, controlled via these two properties, to create effects that emphasize the indicator.
```

Comment:
I don’t think any of these changes are applicable to the iOS or macOS map SDK yet.

Reply:
@1ec5 what do you think is missing / blocking the availability of the source `volatile` flag? GL Native enabled customers setting the flag in the style, and I thought that should flow through to the iOS SDK, particularly since caching is controlled in native

Reply:
Sorry, I overlooked that one. It should work fine, but we should mention it as an option in style JSON. To my knowledge, we haven’t hooked it up to `MGLTileSourceOption` in the runtime styling API. It would be an easy change but it just hasn’t happened yet.